setup.py

```diff
@@ -155,21 +175,7 @@ def initialize_options(self) -> None:
     #
     # 'sqlalchemy-postgres-copy>=0.5,<0.6',
     'pybigquery',
-] + db_dependencies
-
-smart_open_dependencies = [
-    # we rely on exception types from smart_open,
-    # which seem to change in feature releases
-    # without a major version bump
-    'smart_open>=2,<2.1',
-]
-
-aws_dependencies = [
-    'awscli>=1,<2',
-    'boto>=2,<3',
-    'boto3',
-    's3-concat>=0.1.7,<0.2'
-] + smart_open_dependencies
+] + gcs_dependencies + db_dependencies
```

Comment:
Change 2: (I know the diff is hard to read here) - Ensure if someone asks for BigQuery support they also get GCS libraries.